DHL/Entity/GB/ShipmentRequest.php

```diff
@@ -195,5 +207,6 @@ class ShipmentRequest extends Base
             'required' => false,
             'subobject' => true,
         ), 
+        
```

Comment:
You should always review your staged code before committing, and revert any whitespace changes like this. It pollutes the commit log and history. Please revert this line change.